Gopkg.toml

```diff
@@ -28,21 +28,34 @@
 #
 # Kubernetes packages
 #
+
 [[constraint]]
   name = "k8s.io/kubernetes"
-  version = "~1.10"
+  version = "~1.11"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "~7.0"
+  version = "~8.0"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.10.0"
+  version = "kubernetes-1.11.0"
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.10.0"
+  version = "kubernetes-1.11.0"
+
+# vendor/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go:104:16:
+# unknown field 'CaseSensitive' in struct literal of type jsoniter.Config
+[[override]]
+  name = "github.com/json-iterator/go"
+  revision = "f2b4162afba35581b6d4a50d3b8f34e33c144682"
+
+# vendor/k8s.io/client-go/plugin/pkg/client/auth/azure/azure.go:300:25:
+# cannot call non-function spt.Token (type adal.Token)
+[[override]]
```

Comment:
we already have a `[[constraint]]` for this package under `Cloud provider packages` - can you just update the constraint to specify this revision rather than adding an override?

Reply:
I removed the constraint below. I believe this needs to be an override instead of a constraint because we need to override the version used in `k8s.io/client-go`.